jobs/v3/pom.xml

```diff
@@ -54,9 +54,9 @@
       <groupId>com.google.http-client</groupId>
       <artifactId>google-http-client-jackson2</artifactId>
     </dependency>
-    <!-- [END dependencies] -->
+    <!-- [END job_dependencies] -->
 
-    <!-- Test dependencies -->
+    <!-- Test job_dependencies -->
```

Comment:
Just noticed this as well, this update to the comment isn't needed here and it's not accurate. You aren't testing the region tag.

Reply:
Fixed